src/rpc/rpc-client.cpp

```diff
@@ -27,6 +27,11 @@ extern "C" {
 
 namespace {
 
+enum {
+    DOWNLOAD = 0,
+    UPLOAD
```

Comment:
Every name in an enumeration is assigned an integral value that corresponds to its place in the order of the values in the enumeration. By default, the first value is assigned 0, the next one is assigned 1, and so on, but you can explicitly set the value of an enumerator.

https://msdn.microsoft.com/en-us/library/2dzy4k6e.aspx